addons/mail/static/src/components/popover_view/popover_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.PopoverView" owl="1">
+        <div class="o_PopoverView">
+            <t t-if="popoverView and popoverView.target">
```

Comment:
comment on remove check of target (and make it required) was obsolete but not done